tmp/sirius-script-app-bo-pso-inj.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python-sirius
+
+import time as _time
+import numpy as np
+from epics import PV as _PV
+
+
+class PSO:
+    def __init__(self, dim=5):
+        self.pv_ioc_prefix_SOFB = 'BO-Glob:AP-SOFB:'
+        self.pv_ioc_prefix_PosAng = 'TB-Glob:AP-PosAng'
+        self.pv_orbx = _PV(self.pv_ioc_prefix_SOFB + 'SPassOrbX-Mon')  # um
+        self.pv_orby = _PV(self.pv_ioc_prefix_SOFB + 'SPassOrbY-Mon')  # um
+        self.pv_sum = _PV(self.pv_ioc_prefix_SOFB + 'SPassSum-Mon')  # counts
+        self.pv_dx = _PV(self.pv_ioc_prefix_PosAng + 'DeltaPosX-SP')  # mm
+        self.pv_dxl = _PV(self.pv_ioc_prefix_PosAng + 'DeltaAngX-SP')  # mrad
+        self.pv_dy = _PV(self.pv_ioc_prefix_PosAng + 'DeltaPosY-SP')  # mm
+        self.pv_dyl = _PV(self.pv_ioc_prefix_PosAng + 'DeltaAngY-SP')  # mrad
+        self.pv_resetposang = _PV(
+            self.pv_ioc_prefix_PosAng + 'SetNewRefKick-Cmd')
+        self.pv_kckr = _PV('BO-01D:PM-InjKckr:Kick-SP')
+        _time.sleep(2)
```

Comment:
My experience is that simple reads,  `self.pv_resetposang.get()` and `self.pv_kckr.get()`, instead of hardcoded `_time.sleep(2)`, force PV connections at this point of code execution, if providing IOCs are running. If they are not, 2s sleep to allow them to start in the meantime seems arbitrary...

Reply:
ok. temporary script...